docs/io/visualization/how_to_liv_plot.ipynb

```diff
@@ -0,0 +1,419 @@
+{
```

Comment:
<p>We shouldn't allow the user to define narrower bins. The only method we want to allow to increase the resolution is by the user increasing the number of shells in the model. So I would remove the comment above this cell to explicitly mention that you can only re-bin the model to broader bins. And I would also add an exception to the main code that returns an error if the user requests num_bins &gt; the number of shells in the model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>